### ISSUE TITLE
Can't login - Authentication failed and credentials are correct

### ISSUE THREAD

**Issue by JimmyVaneyck** [Esterni/pyracing#99]
I don't know if this is a stupid question but I couldn't login anymore after a maintenance (error from client: "The login POST request was redirected to /failedlogin, indicating an authentication failure. If credentials are correct, check that a captcha is not required by manually visiting members.iracing.com"). 
I looked if captcha was required when I wanted to login on the website and this was required but what should I do to fix this problem so I could use this api for my app again? Thanks.

Edit: 
With the Season 3 release (2022), these endpoints will begin sending the password as a Base64 encoded string (still transmitted via HTTPS) using the following steps.

1. Convert the username (email) to lowercase
2. Concatenate the output from step 1 to the end of the password
3. Create a SHA256 hash of the output from step 2
4. Encode the output from step 3 in Base64
5. Submit the output from step 4 in the password field of the login form

**Comment by smcallis77** [comment]
Hi Jimmy

I am having the same issue as you since the season3 release. Before that, like you, it was working perfectly.  I have tried to follow your fix without success. I may have misinterpreted your solution. To clarify your instructions.

In step 1 & 2, if the username is 'fred@domain.con' and the password is 'iracingpw' then create the string 'iracingpwfred@domain.com'.
In step 3 create the SHA256 hash using the string. I used this site https://passwordsgenerator.net/sha256-hash-generator/
In step 4 encode step 3 output to Base64. I used this site https://www.base64encode.org/
In step 5 not sure what this means. I assume it means provide the username (from step 1) and the password (from the output of step 4) to pyracing.Client()
